src/reference/asciidoc/whats-new.adoc

```diff
@@ -289,3 +289,8 @@ See <<ip>> for more information.
 The `GemfireMetadataStore` now implements `ListenableMetadataStore`, allowing users to listen to cache events by providing `MetadataStoreListener` instances to the store.
 
 See <<gemfire>> for more information.
+
+==== Jdbc Message Channel Store Changes
+
+The `JdbcMessageChannelStore` now provides setter to set `PreparedStatementSetter`, allowing users to easily customize
+`PreparedStatementSetter` in the store.
```

Comment:
In the end need a like to the target chapter:
```
See <<jdbc-message-store-channels>> for more information.
```

Reply:
Ill check and update.